load.php

```diff
@@ -398,6 +394,23 @@ function perflab_maybe_set_object_cache_dropin() {
 		return;
 	}
 
+	/**
+	 * Filters the value of the `object-cache.php` drop-in constant.
+	 *
+	 * This filter should not be used outside of tests.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param int|bool $current_dropin_version The drop-in version as defined by the
+	 *                                         `PERFLAB_OBJECT_CACHE_DROPIN_VERSION` constant.
+	 */
+	$current_dropin_version = apply_filters( 'perflab_object_cache_dropin_version', PERFLAB_OBJECT_CACHE_DROPIN_VERSION );
+
+	// Bail if already placed in the latest version or newer.
+	if ( $current_dropin_version && $current_dropin_version >= PERFLAB_OBJECT_CACHE_DROPIN_LATEST_VERSION ) {
```

Comment:
Will these always be integers? Otherwise `version_compare` would be more robust or future proof

Reply:
Was thinking about that. I went with the simpler numeric comparison given we're just using integers. If you feel strongly about `version_compare()`, we can use that. I guess I've never used `version_compare()` with integers, does that work?

Reply:
It does work, yeah, even though the docs say it expects a string in the form of a typical version number.

I don't feel strongly about that, so feel free to leave as-is

Reply:
Integer comparison seems fine to me. It would be analogous to `$wp_db_version`.